Compiling for 11.04 on jetson calls undefined method: .cudnnOpsInferVersionCheck()

Hi.

I'm compiling cudarc on a Nvidia Jetson AGX Xavier, with CUDA 11.04.

I am getting the following compile errors:
```
| error[E0599]: no method named `cudnnOpsInferVersionCheck` found for reference `&'static cudnn::sys::sys_11040::Lib` in the current scope
|   --> /usr/src/debug/visioncuda/0.1.0.AUTOINC+ced87111db-r0/cargo_home/bitbake/cudarc-0.11.8/src/cudnn/result.rs:55:15
|    |
| 55 |         lib().cudnnOpsInferVersionCheck().result()?;
|    |               ^^^^^^^^^^^^^^^^^^^^^^^^^
|    |
| help: there is a method `cudnnOpsVersionCheck` with a similar name
|    |
| 55 |         lib().cudnnOpsVersionCheck().result()?;
|    |               ~~~~~~~~~~~~~~~~~~~~
```

These errors come from the following section: https://github.com/coreylowman/cudarc/blob/d7ac2b481cb637f7a73f2520ff2d12809285133f/src/cudnn/result.rs#L51

I am not able to find any references to `cudnnOpsInferVersionCheck/cudnnOpsTrainVersionCheck` in the code. Could this be some old API that has slipped trough CI tests or something?

Any insight into this would be greatly appreciated!

Regards, Morten

--- a/src/cudnn/result.rs
+++ b/src/cudnn/result.rs
@@ -46,7 +46,21 @@ pub fn get_cudart_version() -> usize {
 
 /// Runs all *VersionCheck functions.
 pub fn version_check() -> Result<(), CudnnError> {
-    #[cfg(not(any(feature = "cuda-12030", feature = "cuda-12040", feature = "cuda-12050")))]
+    #[cfg(feature = "cuda-11040")]
+    unsafe {
+        lib().cudnnAdvVersionCheck().result()?;
+        lib().cudnnCnnVersionCheck().result()?;
+        lib().cudnnOpsVersionCheck().result()?;
+    }
+    #[cfg(any(
+        feature = "cuda-11050",
+        feature = "cuda-11060",
+        feature = "cuda-11070",
+        feature = "cuda-11080",
+        feature = "cuda-12000",
+        feature = "cuda-12010",
+        feature = "cuda-12020"
+    ))]
     unsafe {
         lib().cudnnAdvInferVersionCheck().result()?;
         lib().cudnnAdvTrainVersionCheck().result()?;
@@ -55,7 +69,7 @@ pub fn version_check() -> Result<(), CudnnError> {
         lib().cudnnOpsInferVersionCheck().result()?;
         lib().cudnnOpsTrainVersionCheck().result()?;
     }
-    #[cfg(feature = "cuda-12030")]
+    #[cfg(any(feature = "cuda-12030", feature = "cuda-12050"))]
     unsafe {
         lib().cudnnAdvVersionCheck().result()?;
         lib().cudnnCnnVersionCheck().result()?;
@@ -63,15 +77,12 @@ pub fn version_check() -> Result<(), CudnnError> {
     }
     #[cfg(feature = "cuda-12040")]
     unsafe {
+        lib().cudnnAdvTrainVersionCheck().result()?;
+        lib().cudnnCnnTrainVersionCheck().result()?;
+        lib().cudnnOpsTrainVersionCheck().result()?;
         lib().cudnnAdvInferVersionCheck().result()?;
         lib().cudnnCnnInferVersionCheck().result()?;
         lib().cudnnOpsInferVersionCheck().result()?;
-    }
-    #[cfg(feature = "cuda-12050")]
-    unsafe {
-        lib().cudnnAdvVersionCheck().result()?;
-        lib().cudnnCnnVersionCheck().result()?;
-        lib().cudnnOpsVersionCheck().result()?;
     }
     Ok(())
 }

